src/test/java/org/jabref/logic/layout/format/EntryTypeFormatterTest.java

```diff
@@ -2,30 +2,30 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class EntryTypeFormatterTest {
 
-    private EntryTypeFormatter formatter;
-
-    @BeforeEach
-    public void setUp() {
-        formatter = new EntryTypeFormatter();
-    }
-
-    @Test
-    public void testCorrectFormatArticle() {
-        assertEquals("Article", formatter.format("article"));
-    }
+    private EntryTypeFormatter formatter = new EntryTypeFormatter();
 
-    @Test
-    public void testCorrectFormatInBook() {
-        assertEquals("InBook", formatter.format("inbook"));
+    @ParameterizedTest
+    @MethodSource("formatTests")
+    void testCorrectFormat(String expectedString, String inputString){
+        assertEquals(expectedString, formatter.format(inputString));
     }
 
-    @Test
-    public void testIncorrectTypeAarticle() {
-        assertEquals("Aarticle", formatter.format("aarticle"));
+    private static Stream<Arguments> formatTests() {
```

Comment:
Overengineered. Revert to the original one. No extra benefit

Reply:
Kind of over engineered. I like the consistency to other tests. Reverting would lead to over-over-engineered :p